TumeKyouen/View/StoneButton.swift

```diff
@@ -16,6 +16,7 @@ class StoneButton: UIButton {
     }
 
     var stoneState: ButtonState = .Blank
+    var delegate: TapDelegate = TumeKyouenDelegate()
```

Comment:
ERROR: [Warning][weak_delegate] Weak Delegate
Delegates should be weak to avoid reference cycles.